.changes/unreleased/NOTES-20231027-085131.yaml

```diff
@@ -0,0 +1,5 @@
+kind: NOTES
```

Comment:
Should this not be a breaking change just like the last node update?
https://github.com/hashicorp/setup-terraform/blob/main/CHANGELOG.md?plain=1#L62-L64

Reply:
Besides this change would require newer runner versions, it should not break anything.